apps/vaults/components/details/VaultActionsTabsWrapper.tsx

```diff
@@ -56,7 +74,11 @@ function	VaultActionsTabsWrapper(): ReactElement {
 					<div className={'relative z-50'}>
 						<Listbox
 							value={currentTab.label}
-							onChange={(): void => onSwitchSelectedOptions()}>
+							onChange={(value): void => {
+								const	newTab = tabs.find((tab): boolean => tab.value === Number(value)) as TTabsOptions;
+								set_currentTab(newTab);
```

Comment:
I think we should stop with castings, and instead address the underlying issue. For example, in this case, `set_currentTab` is not happy to accept an `undefined` so we could either handle the `undefined` value or simply do:
```suggestion
								const	newTab = tabs.find((tab): boolean => tab.value === Number(value));
								if (!newTab) {
									return;
								}
								set_currentTab(newTab);
```
`tabs.find()` might return `undefined` so we should let that be and not override typescript and tell it that we are sure that's never going to be the case if that makes sense